### DISCUSSION TITLE
Is it possible to display current layer numbers?

### DISCUSSION THREAD

**Discussion by AllanMartel** [mriscoc/Ender3V2S1#1212] [Ideas]
I am curious if it is possible to display layer numbers in addition to (or in lieu of) the progress meter?

I am thinking a simple "Printing Layer X / XXX" display. Is this a possibility?

**Comment by mriscoc** [comment]
I do that using a slicer script and M117.

**Comment by AllanMartel** [comment]
Could you elaborate for me please?

**Reply by mriscoc** [reply]
Check: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides#post-processing-scripts

**Reply by AllanMartel** [reply]
Thank you, I started reading this one but obviously didn't get far enough.

**Comment by AllanMartel** [comment]
I have read about the M117 command, but what I cannot determine is how to add this information.

Can you post an example of the script that you use?
